consoles/video_stream.pm

```diff
@@ -225,6 +228,28 @@ sub send_key_event ($self, $key, $press_release_delay) {
       or die "failed to send '$key' input event";
 }
 
+sub type_string ($self, $args) {
+    my $keyboard_device_url = $bmwqemu::vars{GENERAL_HW_PICO_KEYBOARD_URL};
```

Comment:
Likely you could have used different hardware to run that HTTP server on. As far as os-autoinst is converned this is just the URL to *some* HTTP server that'll allows sending keystrokes to the SUT *somehow*. So the variable name doesn't need to be more specific than that:
```suggestion
    my $keyboard_device_url = $bmwqemu::vars{GENERAL_HW_KEYBOARD_URL};
```

Please also add documentation for the newly introduced variable.